src/actions/FirstOpenActions.tsx

```diff
@@ -1,17 +1,27 @@
+import { getAttributionToken } from '@brigad/react-native-adservices'
 import { asNumber, asObject, asOptional, asString, asValue } from 'cleaners'
 import { makeReactNativeDisklet } from 'disklet'
+import { Platform } from 'react-native'
 
 import { FIRST_OPEN } from '../constants/constantSettings'
 import { makeUuid } from '../util/rnUtils'
+import { snooze } from '../util/utils'
 import { getCountryCodeByIp } from './AccountReferralActions'
 
 export const firstOpenDisklet = makeReactNativeDisklet()
 
+const asAppleAdsAttribution = asObject({
+  campaignId: asOptional(asNumber),
+  keywordId: asOptional(asNumber)
+})
+type AppleAdsAttribution = ReturnType<typeof asAppleAdsAttribution>
+
 const asFirstOpenInfo = asObject({
   isFirstOpen: asValue('true', 'false'),
```

Comment:
This should be `asBoolean`.

Reply:
For `isFirstOpen`? I think this was leftover from google analytics and we need to avoid data inconsistencies. If we change this to a boolean, the existing PostHog reports might need to all be updated

Reply:
Oh, shoot! These are strings. Nevermind, I totally missed that. We could clean this up using a `asStringBool` cleaner like this:

```ts
const asStringBool: Cleaner<boolean> = (raw) => {
  if (typeof raw === 'string'){
    if (raw === 'true') return true
    if (raw === 'false') return false
  }
  return asBoolean(raw)
}
```

This would coerce the existing data into a proper boolean on the next open. However, I see that this is out-of-scope for this PR, so let's shelve the idea for now.

Reply:
Yeah in GA4 days, we had trouble using boolean values, so we resorted to strings. We didn't want to change the local disk cache since then, so we had to keep it that way.